spec/xcpretty/formatters/simple_spec.rb

```diff
@@ -20,6 +20,11 @@ module XCPretty
         "> Building Pods/The Spacer [Debug]"
       end
 
+      it "formats analyze target/project/configuration with target" do
+        @formatter.format_analyze_target("The Spacer", "Pods", "Debug").should ==
```

Comment:
Line is too long. [81/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.